deliver/lib/deliver/upload_metadata.rb

```diff
@@ -184,7 +184,7 @@ def load_from_filesystem(options)
           next unless File.exist?(path)
 
           UI.message("Loading '#{path}'...")
-          options[key] ||= {}
+          options[key] = {}
```

Comment:
I think your change is not so good.

According to the documentation, `options[key]` should be hash by languages.

https://github.com/fastlane/fastlane/blob/master/deliver/Deliverfile.md#metadata

```
description({
  'en-US' => "English Description here",
  'de-DE' => "Deutsche Beschreibung hier"
})
```

So, if you got the error, your `Deliverfile` would be wrong.

Your approach can't work fine in some situations.

In your Deliverfile.

```
description({
  'en-US' => "English Description here",
  'de-DE' => "Deutsche Beschreibung hier"
})
```

and there are two metadata for `en-US` and `JP`.

In above situation, these values were merged.
However, your implementation will cause loss of `de-DE`.